package.json

```diff
@@ -7,22 +7,34 @@
     "url": "react-bootstrap/react-bootstrap"
   },
   "homepage": "https://react-bootstrap.github.io/",
+  "main": "lib/index.js",
+  "jsnext:main": "es/index.js",
+  "module": "es/index.js",
   "scripts": {
     "build": "babel-node tools/build-cli.js",
     "docs": "babel-node docs/dev-run.js",
     "docs-build": "babel-node tools/build-cli.js --docs-only",
     "docs-prod": "npm run docs-build && cross-env NODE_ENV=production babel-node docs/server.js",
     "docs-prod-unoptimized": "npm run docs-build -- --dev && cross-env NODE_ENV=production babel-node docs/server.js",
+    "format": "prettier --write '**/*.js' --ignore-path .eslintignore",
     "lint": "eslint src test tools webpack *.js",
+    "precommit": "lint-staged",
     "release": "release",
     "tdd": "karma start",
     "test": "npm run lint && npm run test-browser && npm run test-node",
     "test-browser": "cross-env NODE_ENV=test karma start --single-run",
     "test-node": "mocha --compilers js:babel-register test/server/*Spec.js"
   },
-  "main": "lib/index.js",
-  "jsnext:main": "es/index.js",
-  "module": "es/index.js",
+  "lint-staged": {
+    "*.js": [
+      "eslint --fix",
+      "git add"
+    ]
+  },
+  "prettier": {
+    "printWidth": 80,
```

Comment:
this is the default, no?

Reply:
huh, TIL

Reply:
yeah i just set it to 79 in many places because, uhh, never mind https://prettier.io/docs/en/options.html#print-width